ivy.xml

```diff
@@ -61,6 +61,7 @@
         <dependency org="org.hamcrest" name="hamcrest-library" rev="${hamcrest.version}" conf="test"/>
         <dependency org="org.apache.ant" name="ant-testutil" rev="${apache-ant.version}" conf="test" transitive="false"/>
         <dependency org="org.apache.ant" name="ant-launcher" rev="${apache-ant.version}" conf="test" transitive="false"/>
+        <dependency org="org.apache.ant" name="ant-junit" rev="${apache-ant.version}" conf="test" transitive="false"/>
```

Comment:
why has this dependency been added? I don't see it used.

Reply:
If Ant (launcher and core) is fetched as a dependency, then ant-junit must be retrieved as a dependency, too, in order to run unit tests.

Reply:
to run which unit tests? Ivy's?

Reply:
Yes. Invocation of Ant runs Ivy's build.xml; it's a fancy way of bootstrapping an Ivy build, by the looks of it.

Reply:
strange, thanks. As Ivy uses junit4 you will also need ant-junit4 in this case.

Reply:
Good catch, thanks. BTW, run this way Ant core seems unable to simply find org.apache.tools.ant.taskdefs.optional.junit.JUnitTask on a classpath; why is that? does it need some hint from the launcher?

Reply:
Ah, I get it... the way things work, ant-junit4 must declare a dependency on ant-junit